lib/postgres.go

```diff
@@ -0,0 +1,1414 @@
+package lib
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/dgraph-io/badger/v3"
+	"github.com/go-pg/pg/v10"
+	"github.com/go-pg/pg/v10/orm"
+	"github.com/golang/glog"
+)
+
+type Postgres struct {
+	db *pg.DB
+}
+
+func NewPostgres(db *pg.DB) *Postgres {
+	// Print all queries.
+	//db.AddQueryHook(pgdebug.DebugHook{
+	//	Verbose: true,
+	//})
+
+	return &Postgres{
+		db: db,
+	}
+}
+
+func LogSelect(query *orm.Query) error {
+	selectQuery := orm.NewSelectQuery(query)
+	fmter := orm.NewFormatter().WithModel(selectQuery)
+	queryStr, _ := selectQuery.AppendQuery(fmter, nil)
+	glog.Info(string(queryStr))
```

Comment:
Is it possible we'd ever log something sensitive here? Pretty sure the answer is no, but just flagging. DM me to discuss if so since the logging is definitely useful.

Reply:
Nope!